packages/frontend/src/pages/Tile/components/TableBanner/TableCollaborators.tsx

```diff
@@ -54,7 +59,10 @@ const TableCollabRoleSelect = ({
       >
         {value}
       </MenuButton>
-      <MenuList w={32}>
+      <MenuList w={32} pointerEvents={isEditable ? 'auto' : 'none'}>
```

Comment:
what's the purpose of the `pointerEvents` here ah, cus i thought `MenuList` will never pop out since `MenuButton` already has the `pointerEvents` disabled if not editable

Reply:
u r right